roles/neutron-prerequisites/tasks/main.yml

```diff
@@ -17,20 +17,24 @@
 - include: ubuntu.yml
   when: ansible_distribution == 'Ubuntu'
 
-- name: Create required OVS bridges
-  openvswitch_bridge:
-    bridge: "{{ item }}"
-    state: present
-  with_items:
-    - br-pf9
+- name: check if OVS bridge already exists
```

Comment:
This task is unnecessary. The `openvswitch_bridge` module is idempotent. It will not create the bridge if it already exists.